src/Microsoft.ML.Core/ComponentModel/ComponentCatalog.cs

```diff
@@ -343,7 +343,7 @@ private static bool ShouldSkipPath(string path)
             case "libvw.dll":
             case "matrixinterf.dll":
             case "Microsoft.ML.neuralnetworks.gpucuda.dll":
-            case "Microsoft.ML.mklimports.dll":
+            case "MklImports.dll":
```

Comment:
Obviously doesn’t need to be changed on this PR, but this doesn’t handle OS X and Linux, right?  It only works for Windows.

Reply:
we'd have to do it for all those dlls.